Client/sdk/game/CEntity.h

```diff
@@ -117,4 +117,7 @@ class CEntity
     virtual bool SetBoneRotationQuat(eBone boneId, float x, float y, float z, float w) = 0;
     virtual bool GetBonePosition(eBone boneId, CVector& position) = 0;
     virtual bool SetBonePosition(eBone boneId, const CVector& position) = 0;
+
+    virtual bool IsOnFire() = 0;
+    virtual bool SetOnFire(bool onFire) = 0;
```

Comment:
`const` and maybe `noexcept`?

Reply:
It can't be const because the interfaces aren't const, and I don't want to go beyond the scope of the PR by adding a const GetInterface specifically to the CVehicleSA, CPedSA, and CObjectSA classes. As for noexcept, like I mentioned before, I believe noexcept shouldn't be added to functions that directly call game functions like SetOnFire

Reply:
oh wait haha I already asked about this. Sorry, my bad
(the message got deleted?)

Reply:
No, you replied in the old conversation lol
![image](https://github.com/user-attachments/assets/4750a58f-a390-4289-bdd2-cdf52e534328)

Reply:
bruh I replied here but for some reason it put the comment to previous discussion. Sorry my bad